### PR TITLE
Tests: Update Playwright scripts in root package.json

### PR DESCRIPTION
**Pull request by batbattur**
## Summary
1. Added playwright commands in root `package.json`.
2. Updated the `Tests` section in read me.

## QA notes
Please make sure the Readme makes sense. 

closes: #1253

### CODE REVIEW

**Review by ardelato — DISMISSED** [review]
CR 👍 , nice job @batbattur

File: README.md [hunk 1]
```diff
@@ -93,14 +93,35 @@ Here's an overview of the production setup (the focus is on Next.js, therefore d
...
+
+Also you can provide additional flags/options to run single/multiple tests and with different devices/projects:
 
 ```sh
-cd frontend && pnpm run playwright:run
+pnpm run playwright:run --project="Desktop Chrome" fix_kit         // Runs fix_kit test for desktop chrome headless
+pnpm run playwright:run --project="Mobile Chrome" --headed fix_kit // Runs fix_kit test for mobile chrome headed
 ```
 
+Note: We need to have the dev server running before we run the Playwright tests.
```
Comment (deltuh-vee) [inline README.md:114]:
```suggestion
> ⚠️  We need to have the dev server running before we run the Playwright tests.
```

File: README.md [hunk 2]
```diff
@@ -93,14 +93,35 @@ Here's an overview of the production setup (the focus is on Next.js, therefore d
...
+Luckily, Playwright webserver will start the app automatically if it's not running yet.
+
+This command will run Jest tests:
+
+```sh
+pnpm test
+```
+
+Note: We don't need to have the dev server running before we run the Jest tests.
```
Comment (deltuh-vee) [inline README.md:123]:
```suggestion
> ⚠️ We don't need to have the dev server running before we run the Jest tests.
```

**Review by deltuh-vee — COMMENTED** [review]
I think this is a little more consistent with the rest of the documentation but I'm fine with it as it is.

File: README.md (hunk 2 above)
Comment (batbattur) [reply README.md]: Nice catch @deltuh-vee!

**Comment by deltuh-vee** [comment]
QA 🎬 
Readme looks good.

**Review by ardelato — APPROVED** [review]
CR 👍

**Pull request merged**